src/main/java/jpa/domain/Station.java

```diff
@@ -0,0 +1,34 @@
+package jpa.domain;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "station")
+public class Station extends BaseEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@Column(name = "name", nullable = false, unique = true, length = 255)
+	private String name;
+
+	public Station() {
```

Comment:
Default constructor가 public으로 열릴 필요는 없지 않을까요?

```suggestion
	protected Station() {
```

아래 링크의 `4.1.1. Implement a no-argument constructor` 항목을 한번 참고해보세요.

https://docs.jboss.org/hibernate/core/3.5/reference/en/html/persistent-classes.html

```
Cat has a no-argument constructor. All persistent classes must have a default constructor (which can be non-public) so that Hibernate can instantiate them using Constructor.newInstance(). It is recommended that you have a default constructor with at least package visibility for runtime proxy generation in Hibernate.
````

Reply:
주신 링크 잘 읽어보았습니다!
좀 더 꼼꼼히 접근제한자를 설정하여야 하겠습니다.
감사합니다.